src/Alert.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+
+const alertTypes = ['success', 'info', 'warning', 'danger'];
+
+const propTypes = {
+  color: React.PropTypes.oneOf(alertTypes),
```

Comment:
@TheSharpieOne yo! thanks for the input. i'll talk to @eddywashere and see what we can do to take your use case into account. thanks a billion for the feedback. : )